teamA/crud-express/heeyeon/routes/userRouter.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const router = express.Router();
+
+const userController = require('../controllers/userController');
+router.post('/signup', userController.signUp);
+// router.post('/signin', userController.signIn);

```

Comment:
- 개발 시 작성한 주석은 지워주시고, 줄바꿈해주세요 !
```suggestion
const userController = require('../controllers/userController');

router.post('/signup', userController.signUp);
```